app/tasks/job_attributes_fetcher.rb

```diff
@@ -42,14 +42,13 @@ def fetch_application_question_set
   # i.e : autocomplete & mandatory location
   # Returns true if no such question is found, indicating the user can apply with Cheddar.
   def apply_with_cheddar
-    return false unless form_structure.dig(:core_questions, :questions)
+    return false unless form_structure.dig('core_questions', 'questions')
```

Comment:
Why are we accessing off strings rather than symbols now? What is the rule we're applying in general?

Reply:
I don't know. I think i am loosing sym after saving into the database. It is fix because we are in a sprint.